support-workers/src/main/scala/com/gu/zuora/ProductSubscriptionBuilders.scala

```diff
@@ -128,17 +118,21 @@ object ProductSubscriptionBuilders {
         case Failure(e) => throw new BadRequestException(s"First delivery date was not provided. It is required for a Guardian Weekly subscription.", e)
       }
 
-      val productRatePlanId = getProductRatePlanId(catalog.GuardianWeekly, guardianWeekly, isTestUser)
+      val recurringProductRatePlanId = validateRatePlan(guardianWeekly.productRatePlan(getTouchPointEnvironment(isTestUser)), guardianWeekly.describe)
```

Comment:
Because a 6 for 6 sub in Zuora is made up of two separate rate plans we need to get the id for both, one for the standard recurring part and one for the promotional part